### PR TITLE
LEDE-2613 Initial Styling Cleanup of Newsletter Blocks

### PR DESCRIPTION
**Pull request by cahdeemer**
## Summary
Cleans up styles generally, addresses some layout issues with the `two-up-post` block and the close button. Focus was on ensuring consistent use of fonts families and font sizes.

## Notes for reviewers
* Version bump

## Ticket
https://alleyinteractive.atlassian.net/browse/LEDE-2613

### CODE REVIEW

File: blocks/two-up-post/index.scss [hunk 1]
```diff
@@ -5,15 +5,24 @@
...
+  div[data-title="Newsletter Single Post"] {
+    .block-editor-inner-blocks{
+      display: flex;
 
-    > div {
-      width: 50%;
+      // stylelint-disable selector-max-compound-selectors
+      .block-editor-block-list__layout {
+      display: flex;
+      flex-direction: column;
```
Comment (mogmarsh) [inline blocks/two-up-post/index.scss]: indent?

File: blocks/two-up-post/style.scss [hunk 2]
```diff
@@ -21,7 +21,7 @@
   div.wp-block-wp-newsletter-builder-post {
     max-width: 300px;
     padding: 0 10px;
-    width: 50%;
+    flex: 1 1 50%;
```
Comment (mogmarsh) [inline blocks/two-up-post/style.scss]: this probably won't work in the email client.

Comment (mogmarsh) [reply blocks/two-up-post/style.scss]: this may not show up there either... if so, ignore me.

**Review by mogmarsh — APPROVED** [review]
👍 this looks good... just a reminder that any css that will be shown on the front end (is used when you preview) needs to be email client friendly. I flagged one that may be a concern. https://www.caniemail.com/ might be helpful.

File: blocks/two-up-post/style.scss (hunk 2 above)
Comment (cahdeemer) [reply blocks/two-up-post/style.scss]: Email styling is literally the worst

File: blocks/two-up-post/index.scss (hunk 1 above)
Comment (cahdeemer) [reply blocks/two-up-post/index.scss]: fixed

File: blocks/two-up-post/style.scss (hunk 2 above)
Comment (cahdeemer) [reply blocks/two-up-post/style.scss]: It seems to show up, so I reverted back to the`width` property

**Pull request merged**